src/ipc-handlers.ts

```diff
@@ -153,7 +153,7 @@ export async function createSite(
 	enableHttps?: boolean,
 	siteId?: string
 ): Promise< SiteDetails[] > {
-	const userData = await loadUserData();
+	const userData = await loadUserData( true );
```

Comment:
The `createSite` function can probably take more than 1000ms to execute, so maybe we need to hold the lockfile for longer in this case.